ci/build_container/build_container_ubuntu.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # Setup basic requirements and install them.
 apt-get update
 apt-get install -y wget software-properties-common make cmake git python python-pip \
-  bc libtool automake zip time golang g++ gdb strace
+  bc libtool automake zip time golang g++ gdb strace python-sphinx
```

Comment:
Should we use the same venv that we currently use in the main docs build?

Reply:
My plan was to do this by copying `docs/requirements.txt` to the same place in `data-plane-api`. Or, do you want to set this up in the base image?

Reply:
The venv includes sphinx. Do we need this change then?